Xamarin.Forms.Core/MultiPage.cs

```diff
@@ -199,22 +199,24 @@ void OnChildrenChanged(object sender, NotifyCollectionChangedEventArgs e)
 
 		void OnTemplatedItemsChanged(object sender, NotifyCollectionChangedEventArgs e)
 		{
+			var pageController = ((IPageController)this);
```

Comment:
Might consider expanding the idea an making a private property that preforms the cast: `IPageController Controller => this`